templates/config.json

```diff
@@ -282,6 +297,9 @@
       "id": "facility",
       "file": "policies/facility.md",
       "name": "Facility Access and Physical Security",
+      "instructions": [
+        "Do you store any ePHI?"
```

Comment:
```suggestion
        "Do you store any sensitive data such as PII or PHI physically at office locations?",
        "Do you operate or maintain your own data center?"
```

Reply:
Here's another situation where [the Policy](https://j1dev.apps.us.jupiterone.io/policies/facility/cp-physical) makes reference to ePHI. Should we generalize the language of Policies? @tanggk

Reply:
Removed instructions from policy